recipes/glew/run_test.bat

```diff
@@ -0,0 +1,7 @@
+conda env create -f %RECIPE_DIR%\test\test-environment-windows.yml
```

Comment:
Unfortunately no.
I tried doing that, however, on Windows, the `cl.exe` executable is not set. I've pinged the conda-forge gitter channel with this message:

> Hi all, any Windows experts?
> How can I put cl.exe in the PATH during the testing phase? I want to compile a test example during the run_test.bat execution, but I get the following error (from cmake):
>   The CMAKE_CXX_COMPILER:
>     cl
>   is not a full path and was not found in the PATH
> My meta.yaml has:
> test:
>   requires:
>     - cmake
>     - toolchain
>     - vc 9  # [win and py27]
>     - vc 10  # [win and py34]
>     - vc 14  # [win and py35]

Which I discovered how to fix:

> I believe I understand the error now.
> conda-build does not have support for this (at least in the version that conda-forge uses, I can't > answer about conda-build 2).
> The [xtensor feedstock](https://github.com/conda-forge/xtensor-> feedstock/blob/master/recipe/run_test.bat) has solved the problem.
> Others simply skip this ([sundials-feedstock](https://github.com/conda-forge/sundials-feedstock/blob/master/recipe/run_test.bat)).